charts/simple-app/values.yaml

```diff
@@ -143,55 +147,83 @@ securityContext: {}
 
 service:
   type: ClusterIP
-  port: 80
```

Comment:
Why did you remove this?

Reply:
This was a misleading setting - it was only being used by the test framework.